src/reducers/testingMode.js

```diff
@@ -0,0 +1,27 @@
+import config from 'react-native-config';
+
+import { MAKE_STEP, RESET_STEPS } from '../actions/testingMode';
+
+const stepsCountToToggle = 5;
+
+const initialState = {
+  isActive: false,
+  stepsLeftToToggle: stepsCountToToggle,
+};
+
+export default function (state = initialState, action) {
+  switch (action.type) {
+    case MAKE_STEP:
+      if (config.PRODUCTION === 'true') return state;
+
+      if (state.stepsLeftToToggle === 1) {
+        return { ...state, stepsLeftToToggle: stepsCountToToggle, isActive: !state.isActive };
+      }
+      return { ...state, stepsLeftToToggle: state.stepsLeftToToggle - 1 };
+    case RESET_STEPS:
```

Comment:
Don't we here need to check as well for production?

Reply:
You're right, moved check into the beginning of reducer.